src/components/backupWallet/ExportWalletData.js

```diff
@@ -24,8 +25,15 @@ import config from '../../config/config'
 // assets
 import unknownProfile from '../../assets/unknownProfile.svg'
 import FuseLogo from '../../assets/ExportWallet/FuseLogo.svg'
+import ExportWarningPopup from './ExportWarningPopup'
 
-const web3ProviderUrl = GoodWallet.networkId && config.ethereum[GoodWallet.networkId].httpWeb3provider
+if (Platform.OS === 'web') {
```

Comment:
this could be removed - prefetching svgs in react-native creates uncaught promise errors, we have had some of these in the past